met-api/migrations/versions/47fc88fe0477_video_widget.py

```diff
@@ -0,0 +1,52 @@
+"""Add video widget
+
+Revision ID: 47fc88fe0477
+Revises: b3b5c66cea4b
+Create Date: 2023-07-11 10:44:35.980432
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '47fc88fe0477'
+down_revision = 'b3b5c66cea4b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('widget_video',
+    sa.Column('created_date', sa.DateTime(), nullable=False),
+    sa.Column('updated_date', sa.DateTime(), nullable=True),
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('widget_id', sa.Integer(), nullable=True),
+    sa.Column('engagement_id', sa.Integer(), nullable=True),
+    sa.Column('video_url', sa.String(length=255), nullable=False),
+    sa.Column('description', sa.Text(), nullable=True),
+    sa.Column('created_by', sa.String(length=50), nullable=True),
+    sa.Column('updated_by', sa.String(length=50), nullable=True),
+    sa.ForeignKeyConstraint(['engagement_id'], ['engagement.id'], ondelete='CASCADE'),
+    sa.ForeignKeyConstraint(['widget_id'], ['widget.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id')
+    )
+    widget_type_table = sa.table('widget_type',
+        sa.Column('id', sa.Integer),
+        sa.Column('name', sa.String),
+        sa.Column('description', sa.String))
+
+    op.bulk_insert(widget_type_table, [
+        {'id': 7, 'name': 'Video', 'description': 'Add a link to a hosted video and link preview'}
```

Comment:
in one of the alembic scripts , i haev a better of handling the increment numbers.. can you look up that?
basically it reads value from the sequnce and does a plus one

Reply:
That's definitely a smarter way to do it. I just choose to hard code 7 here because it is hard coded in the front end:
![image](https://github.com/bcgov/met-public/assets/91914654/48d43ac2-2b1b-47e1-82e7-250fe68276ad)

And types are hard coded in the backend in the constants in WidgetType which is used for the tests. I wasn't comfortable leaving it up for a calculation although it will reach the same result of 7. What do you think

![image](https://github.com/bcgov/met-public/assets/91914654/4e5a55cb-a4e5-4591-82cd-24dbc239df9b)